src/snowflake/snowpark/functions.py

```diff
@@ -9538,9 +9537,6 @@ def sproc(
         2
     """
 
-    if _emit_ast:
-        raise NotImplementedError("TODO 1675257: support sprocs for AST.")
```

Comment:
Did you search for `"1675257"`? No more TODO referencing this ticket in the source code?

Reply:
Yup! I haven't seen anything else referencing this ticket.

Reply:
ok, perfect!